### PR TITLE
[feat] Pomodoro timer

### PR DESCRIPTION
**Pull request by Hamdrive**
## Overview

Successfully created the pomodoro page for a specific task. Implemented the use of [react-circle-progressbar](https://www.npmjs.com/package/react-circular-progressbar) to display the main feature (aka the countdown timer)
Features:
1. User creates task on task page, and on click is redirected to pomodoro page with all times and data loaded
2. Start/Pause/restart functionality ensuring time management is maintained
3. Two modes: focus and break
4. Once a mode is complete, automatically the next mode is started
5. Implementation of setInterval to handle countdown and useRef  hook to pass values that do not trigger unnecessary re-renders

## Deployment

https://drivers-pomodoro-git-pomodoro-hamdrive.vercel.app/tasks

## Preview
image
image
image

### CODE REVIEW

File: src/components/Modal/Modal.jsx [hunk 1]
```diff
@@ -97,21 +104,43 @@ export function Modal({ toggleModal, setTasks, tasks, modal }) {
...
             </datalist>
           </div>
-          <footer className={`${styles.modal__footer}`}>
-            <button
-              onClick={toggleModal}
-              className={`btn btn-md mx-sm ${styles.outline}`}
+          <div className="input-section">
+            <label htmlFor="input" className="form-input input-required">
+              Break duration
+            </label>
```
Comment (junaidshaikh-js) [inline src/components/Modal/Modal.jsx]: For which input element you have used this label? 

I can't find any input element with id `input`

**Review by junaidshaikh-js — APPROVED** [review]
Great work done! 

Good use of useRef hook, loved how you divided the components into smaller parts and separated the functionality of each component. Good to merge.

**Review by AnuragSahu11 — APPROVED** [review]
Good Ui , proper variable names, different functions for different tasks, you can merge

File: src/components/Modal/Modal.jsx (hunk 1 above)
Comment (Hamdrive) [reply src/components/Modal/Modal.jsx]: Yes that is correct, I had created the input and labels but did not update the `htmlFor` and `id` to link
Thanks for catching that!

**Comment by Hamdrive** [comment]
Pull Request successfully reviewed by two members, changes have been suggested and successfully implemented.
Thus no issues faced and can be successfully merged to dev branch and can be closed

**Pull request merged**